config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -312,6 +234,42 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
+- cron: '1 8 * * 6' # Run at 00:01PST (8:01 UTC) on Sat
+  name: ci-kubernetes-e2e-gke-scale-performance
+  tags:
+  - "perfDashPrefix: gke-5000Nodes"
+  - "perfDashJobType: performance"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=1080
```

Comment:
I'm not 100% sure if it will be enough because there are different qps limits in GKE - but from back-of-envelope calculations it should fit. We can adjust if needed later.

Reply:
Sounds good, will adjust if it fails.